### PR TITLE
#3 creating manual for developement process

### PR DESCRIPTION
**Pull request by KarolAdam**
plis CR

### CODE REVIEW

**Review by IggyTRPD — APPROVED** [review]
👍

**Review by PatrycjaPaczkowska — APPROVED** [review]
ok

**Review by bartgalicki — APPROVED** [review]

File: testkonfliktu.md [hunk 1]
```diff
@@ -1 +1 @@
-abecadło
\ No newline at end of file
+placki
```
Comment (kamilponiatowski) [inline testkonfliktu.md:1]: wole ciasto

**Review by IggyTRPD — APPROVED** [review]
🥇

**Review by kamilponiatowski — CHANGES_REQUESTED** [review]

**Pull request merged**